experiments/ClimaEarth/cli_options.jl

```diff
@@ -85,8 +85,8 @@ function argparse_settings()
         default = nothing
         "--restart_t"
```

Comment:
I can't comment on that line, but could you also delete "hourly" from the description for `checkpoint_dt` in line 78 please?